keyboards/viktus/vkr94/readme.md

```diff
@@ -0,0 +1,27 @@
+# Viktus VKR 94
+
+![viktus/vkr94]()
```

Comment:
Missing link?

Reply:
I didn't have an actual link to insert, but wasn't sure if I was able to delete that line without it breaking something by accident.